### DISCUSSION TITLE
FYI: stubs are being added to typeshed

### DISCUSSION THREAD

**Discussion by richardnias** [stchris/untangle#119] [General]
Hi,

I just wanted to write here to keep the maintainers aware that I've opened a PR to add a `types-untangle` package: python/typeshed#9402. I haven't spent a lot of time with the internals of untangle so there might be mistakes, but for my purposes it works (mypy passes in my project when using the stub file) so I thought I'd share it.

Thanks for the library!

**Comment by stchris** [comment]
Thank you so much, @richardnias ! This makes working with `untangle` a lot more pleasant. I would like to add this to the README (and mention you as the maintainer of the types package if that's ok?).
